spyder/plugins/maininterpreter/confpage.py

```diff
@@ -158,9 +159,41 @@ def setup_page(self):
         umr_layout.addWidget(umr_namelist_btn)
         umr_group.setLayout(umr_layout)
 
+        # Path anaconda executor
+        conda_group = QGroupBox(_("Conda executable"))
+        conda_layout = QVBoxLayout()
+
+        custom_conda_check = self.create_checkbox(
+            _("Set the path to a custom Conda, Mamba or Micromamba executable"),
+            "custom_conda"
+        )
```

Comment:
Make the checkbox text itself more correct, clear, concise and appropriate for the context, and add a tip with an explanation for why a user might need to set this.

```suggestion
            _("Use custom Conda/Mamba/Micromamba executable"),
            "custom_conda",
            tip=_(
                "Use the specified Conda, Mamba or Micromamba instead of "
                "finding the executable from the interpreter path. "
                "Required if using a custom Conda prefix with a Conda/Mamba "
                "installed at a non-standard location."
            ),
        )
```